group_vars/webapp/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+# App
+app_user: dosomething
+app_host: www.dosomething.org
```

Comment:
@sergii-tkachenko No, but unfortunately, due to some symlinking in the files path that links there, it has to stay that name for the moment. I'll create a ticket to figure out a way around this later, but for now, this will get our env up and running.